rednotebook/gui/search.py

```diff
@@ -79,6 +87,36 @@ def search(self, search_text):
             self.entry.grab_focus()
 
 
+class ReplaceBox(Gtk.Box):
+    def __init__(self, main_window, **properties):
+        super().__init__(**properties)
+
+        self.old_data = ""
+
+        self.journal = main_window.journal
+
+        self.text_field = Gtk.Entry()
+        self.text_field.set_placeholder_text("Replace")
+        self.text_field.connect("activate", self.on_entry_activated)
+        self.text_field.show()
+
+        self.pack_start(self.text_field, True, True, 0)
+        self.set_orientation(Gtk.Orientation.HORIZONTAL)
+
+    def on_entry_activated(self, _):
+        """Called when the user hits enter."""
+
+        new_data = self.text_field.get_text()
+        if not new_data or new_data == self.old_data:
```

Comment:
```suggestion
        if new_data == self.old_data:
```
This would allow removing strings of text (without replacing them). Or did you want to prevent this?